Source/WebCore/rendering/RenderLayerCompositor.cpp

```diff
@@ -4481,6 +4477,20 @@ void RenderLayerCompositor::rootBackgroundColorOrTransparencyChanged()
     rootLayerConfigurationChanged();
 }
 
+#if HAVE(RUBBER_BANDING)
+void RenderLayerCompositor::updateSizeAndPositionForOverhangAreaLayer()
+{
+    if (!m_layerForOverhangAreas)
+        return;
+
+    float topContentInset = m_renderView.frameView().topContentInset();
+    IntSize overhangAreaSize = m_renderView.frameView().frameRect().size();
+    overhangAreaSize.setHeight(overhangAreaSize.height() - topContentInset);
```

Comment:
`overhangAreaSize.contract(0, topContentInset)` followed by `clampNegativeToZero()` ?

Reply:
Reads better, I'll do that.